### ISSUE TITLE
CLI crashes for urls under Windows and Python < 3.8 

### ISSUE THREAD

**Issue by chr1st1ank** [kiwicom/schemathesis#399]
When testing this nice module I ran into the following issue:

The error is fairly easy to produce (under Python 3.7):
```
(template-webservice) C:\Users\christiank\PycharmProjects\Template_Webservice>schemathesis run http://localhost:8080/openapi.json
Traceback (most recent call last):
  File "c:\users\christiank\appdata\local\continuum\miniconda3\envs\template-webservice\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\users\christiank\appdata\local\continuum\miniconda3\envs\template-webservice\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\christiank\AppData\Local\Continuum\miniconda3\envs\template-webservice\Scripts\schemathesis.exe\__main__.py", line 9, in <module>
  File "c:\users\christiank\appdata\local\continuum\miniconda3\envs\template-webservice\lib\site-packages\click\core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "c:\users\christiank\appdata\local\continuum\miniconda3\envs\template-webservice\lib\site-packages\click\core.py", line 717, in main
    rv = self.invoke(ctx)
  File "c:\users\christiank\appdata\local\continuum\miniconda3\envs\template-webservice\lib\site-packages\click\core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "c:\users\christiank\appdata\local\continuum\miniconda3\envs\template-webservice\lib\site-packages\click\core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\users\christiank\appdata\local\continuum\miniconda3\envs\template-webservice\lib\site-packages\click\core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "c:\users\christiank\appdata\local\continuum\miniconda3\envs\template-webservice\lib\site-packages\schemathesis\cli\__init__.py", line 196, in run
    if pathlib.Path(schema).is_file():
  File "c:\users\christiank\appdata\local\continuum\miniconda3\envs\template-webservice\lib\pathlib.py", line 1372, in is_file
    return S_ISREG(self.stat().st_mode)
  File "c:\users\christiank\appdata\local\continuum\miniconda3\envs\template-webservice\lib\pathlib.py", line 1168, in stat
    return self._accessor.stat(self)
OSError: [WinError 123] Die Syntax für den Dateinamen, Verzeichnisnamen oder die Datenträgerbezeichnung ist falsch: 'http:\\localhost:8080\\openapi.json'
```

The reason is this line in the code:
https://github.com/kiwicom/schemathesis/blob/f2e160d56c1fdce9eac7fee5875b209c8944f54a/src/schemathesis/cli/__init__.py#L196

From to the [Python docs for pathlib](https://docs.python.org/3/library/pathlib.html):

> Changed in version 3.8: exists(), is_dir(), is_file(), is_mount(), is_symlink(), is_block_device(), is_char_device(), is_fifo(), is_socket() now return False instead of raising an exception for paths that contain characters unrepresentable at the OS level.

So it seems necessary to either move away from is_file() or to add a try-catch-block

**Comment by Stranger6667** [comment]
Hello @chr1st1ank !
Thank you for your report and a PR that fixed the problem :) I will take a look during the weekend.

**Comment by Stranger6667** [comment]
Actually we handle this problem in field validators, with try/except, so I reused existing approach and updated your PR :) Thank you very much again for bringing this issue :) We definitely need to add tests on different platforms.

The fix is released in `0,24.1` https://pypi.org/project/schemathesis/0.24.1/

Let us know if there will be any problems with the new release
